src/modules/todoList.js

```diff
@@ -0,0 +1,88 @@
+import { myInput, todoList } from './elements.js';
+import appendTodoList from './appendToDoList.js';
+
+class TODO {
+  constructor() {
+    this.index = 0;
+    this.completed = false;
+    this.description = '';
+    this.todos = [];
+  }
+
+  addTodo() {
+    this.todos = this.readTodo();
+    this.description = myInput.value;
+    this.index = this.todos.length;
+    this.todos.push({
+      index: this.index,
+      description: this.description,
+      completed: this.completed,
+    });
+    this.storeTodo();
+    appendTodoList({
+      index: this.index,
+      description: this.description,
+      completed: this.completed,
+    });
```

Comment:
- [ ] It would be better if the index of the task is set to the value of the new array length. (i.e. if a 5th task is added to the list, the index of that task should equal 5). Kindly fix this minor issue. Currently, your first task has the value of 0 and should be 1. 

😄 👍